tolstoy/README.md

```diff
@@ -0,0 +1,135 @@
+# Tolstoy, a Mentat Sync implementation
+
+## Current state
+This work is partially a proof-of-concept, partially an alpha implementation of how a generic sync might operate on top of a log-oriented storage layer a la Mentat.
+
+## Overview
+### Very briefly
+Tolstoy will synchronize a local Mentat database against a remote server, modifying local state if necessary, and uploading changes to the server if necessary. Schema additions are allowed (adding vocabulary). Schema mutations are currently not implemented (changing vocabulary). Mentat's core schema must be the same.
```

Comment:
nit: "[must be the same] on all participating clients (i.e., core schema alterations are unsupported)."